MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include services/utils/requirements.txt
+include services/data/requirements.txt
```

Comment:
these two files got deleted in the dependency unpinning commit.

Reply:
removed